### PR TITLE
[DRAFT] Readds space travel

### PR DESCRIPTION
**Pull request by Mat05usz**
## About The Pull Request

What title says, it readds space travel for people that don't have shuttles. Ever wanted to just grab a spacesuit and discover the space like it used to be? That's the way! 

It's still very much a work in progress, there is some stuff that will be tested, or removed after testing. What I want is feedback regarding the travel itself.

What it adds is an ability to travel between z levels for everything. Shit that's considered trash (basically not alive anything) will be sent to a trash z level, which is one of the asteroid. If you are alive, you will be sent to a z level that can be generated. 

Currently the cost for travelling is pretty low, it's distance/100 (its low for now, but like travelling to neareset asteroid will cost your regular oxy tanks to lose about half of moles). So like for a pretty far away objective you need like 4/5 big oxygen internal tanks.

As for the time cost, its minimum 10 seconds maximum 90, and in between depends on the distance. Time to generate ruins counts towards the travel time.

I'll appreciate any ideas.

Basically how travel now works is that: O2 moles needed for travel is distance / 100. (I'll provide more detailed numbers later).

## Why It's Good For The Game

Adding proper space travel that doesn't require shuttles is neat

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
add: Later
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
Can you explain what exactly this is? What do you mean by space travel without shuttles? How does one actually move z levels?

**Comment by FortSumter** [comment]
Idk about this. It would be nice to have in general but I think the specific use for this should be to get at the exploration crew if you're antag with an assassination objective

**Comment by dwasint** [comment]
i'd def do distance based, as to not kill the mood if we ever decide to add long long range missions that are like 5 minutes away.

**Comment by NerdwardIreland** [comment]
I agree, this gives something for the curator to do with his fuckin dumb ass space kit and actually makes spacing bodies viable, good idea.

**Comment by Nosnek199** [comment]
If it is not out of scope with the PR, perhaps any space ruin map on the ruin list should have a gps signal in it? Could make identifying whether or not the zlevel you are on is a junk level or not (completely empty space)

**Comment by Nosnek199** [comment]
yeah that'll probably work

**Comment by dwasint** [comment]
No you can't do this to me.

**Comment by ike709** [comment]
Stale, ask for reopen if you're still working on this.
